targets/TARGET_ONSEMI/TARGET_NCS36510/gpio_api.c

```diff
@@ -143,7 +143,9 @@ void gpio_mode(gpio_t *obj, PinMode mode)
 void gpio_dir(gpio_t *obj, PinDirection direction)
 {
     /* Enable the GPIO clock */
-    CLOCK_ENABLE(CLOCK_GPIO);
+    if(!CLOCK_IS_ENABLED(CLOCK_GPIO)) {
+        CLOCK_ENABLE(CLOCK_GPIO);
+    }
```

Comment:
The GPIO disable in other drivers was also part of a rebase and automerge gone horribly wrong. I hope it is reflected in my latest checkin and pull request. Please let me know.